From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Mon, 18 Apr 2016 17:24:25 +0200
Subject: [PATCH] Separate armband repo-comp from upstream RPM repos.

Instead of using linux.enea.com for ALL RPM packages for CentOS7,
by overriding FUEL_MIRROR in the Makefile, we switched to using
EXTRA_RPM_REPOS variable to point to armband RPM repo-component.

This way, the bulk of RPM packages will be fetched from upstream,
instead of our mirror.

This requires updating ks.cfg{,.orig} with the new armband repo.
---
 build/f_isoroot/f_kscfg/ks.cfg      | 2 +-
 build/f_isoroot/f_kscfg/ks.cfg.orig | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/f_isoroot/f_kscfg/ks.cfg b/build/f_isoroot/f_kscfg/ks.cfg
index 30bad1d..b58f98b 100644
--- a/build/f_isoroot/f_kscfg/ks.cfg
+++ b/build/f_isoroot/f_kscfg/ks.cfg
@@ -14,7 +14,7 @@ unsupported_hardware
 # SKIP CONFIGURING X
 skipx
 repo --name="mos-centos" --baseurl=file:///run/install/repo/mos-centos/ --cost=100
-
+repo --name="armband" --baseurl=file:///run/install/repo/extra-repos/armband --cost=10
 # NEVER ever place zerombr here, it breaks automated installation
 %include /tmp/bootloader.ks
 %include /tmp/partition.ks
diff --git a/build/f_isoroot/f_kscfg/ks.cfg.orig b/build/f_isoroot/f_kscfg/ks.cfg.orig
index f8a43da..5f541a4 100755
--- a/build/f_isoroot/f_kscfg/ks.cfg.orig
+++ b/build/f_isoroot/f_kscfg/ks.cfg.orig
@@ -14,7 +14,7 @@ unsupported_hardware
 # SKIP CONFIGURING X
 skipx
 repo --name="mos-centos" --baseurl=file:///run/install/repo/mos-centos/ --cost=100
-
+repo --name="armband" --baseurl=file:///run/install/repo/extra-repos/armband --cost=10
 # NEVER ever place zerombr here, it breaks automated installation
 %include /tmp/bootloader.ks
 %include /tmp/partition.ks
